src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string | null;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyPresser);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyPresser);
+  }
+
+  keyPresser = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
```

Comment:
I think it is better to write a type of function result. Even if function return nothing